packages/grid/x-data-grid/src/locales/roRO.ts

```diff
@@ -0,0 +1,148 @@
+import { roRO as roROCore } from '@mui/material/locale';
+import { GridLocaleText } from '../models/api/gridLocaleTextApi';
+import { getGridLocalization, Localization } from '../utils/getGridLocalization';
+
+const roROGrid: Partial<GridLocaleText> = {
+  // Root
+  noRowsLabel: 'Lipsă date',
+  noResultsOverlayLabel: 'Nu au fost găsite rezultate.',
+  errorOverlayDefaultLabel: 'A apărut o eroare neașteptată.',
+
+  // Density selector toolbar button text
+  toolbarDensity: 'Înălțime rând',
+  toolbarDensityLabel: 'Înălțime rând',
+  toolbarDensityCompact: 'Compact',
+  toolbarDensityStandard: 'Standard',
+  toolbarDensityComfortable: 'Lat',
+
+  // Columns selector toolbar button text
+  toolbarColumns: 'Coloane',
+  toolbarColumnsLabel: 'Afișează selecție coloane',
+
+  // Filters toolbar button text
+  toolbarFilters: 'Filtru',
+  toolbarFiltersLabel: 'Afișează filtru',
+  toolbarFiltersTooltipHide: 'Ascunde filtru',
+  toolbarFiltersTooltipShow: 'Afișează filtru',
+  toolbarFiltersTooltipActive: (count) =>
+    count !== 1 ? `${count} filtru activ` : `${count} filtru activ`,
+
+  // Quick filter toolbar field
+  // toolbarQuickFilterPlaceholder: 'Search...',
+  // toolbarQuickFilterLabel: 'Search',
+  // toolbarQuickFilterDeleteIconLabel: 'Clear',
```

Comment:
Do you mind translating these keys too? It's used in https://mui.com/x/react-data-grid/filtering/#quick-filter